### DISCUSSION TITLE
Help, The performance optimization is ineffective.

### DISCUSSION THREAD

**Discussion by hushao711** [Nebukam/com.nebukam.orca#6] [Q&A] [answered]
Hi，I have downloaded your ORCA, but the performance optimization is ineffective. In the example named ORCASetup , the time consumption for 1000 units is 44ms.

**Comment by Nebukam** [comment]
Hi there! Sadly without any more details, there's not much help I can offer.

There are plenty of things than can degrade performance in a dev environment: is Burst compiler enabled? Do you have jobs precise stack trace/debugging enabled? What's your hardware? What's your version of Unity?

**Comment by Nebukam** [comment, marked as answer]
I can confirm that ORCASetup with **Burst Compile enabled (and Job debugging enabled, so that's a hit on perfs)**, I get consistently <4ms with 1000 agents:

![image](https://github.com/Nebukam/com.nebukam.orca/assets/5935407/613fd351-3b3d-40b3-96fd-5d71a279b532)

The bulk of the job itself is hovering 10-12ms total.

With Burst compilation off however I get horrendous perf (~80ms ahah), please check out this link to know how to enable burst : [Burst User Guide](https://docs.unity3d.com/Packages/com.unity.burst@0.2/manual/index.html) and let me know if it helps :)

**Comment by Nebukam** [comment]
Hello @hushao711 any news on this?
